packages/wonder-stuff-render-environment-jsdom/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "publishConfig": {
+        "access": "public"
+    },
+    "engines": {
+        "node": ">=16"
+    },
+    "name": "@khanacademy/wonder-stuff-render-environment-jsdom",
+    "version": "0.0.1",
+    "description": "Render environment to use with the Wonder Stuff Render Server",
+    "module": "dist/es/index.js",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "scripts": {
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
```

Comment:
**question:** what does this do?

Reply:
It means that you can run `yarn test` from inside the specific package folder. Makes it easier to focus dev on a single package and have a CLI running at the root of that package.